### PR TITLE
Adding city/country columns to Article model, adding rspec validation…

### PR DESCRIPTION
**Pull request by sethnejame**
Chore: Add City/Country columns to Article Model

### User Story

```
N/A
```
### Feature description
This relates to the geolocation feature. . .the backend model Article needs a city/country column in order for it to work.
PT Link: https://www.pivotaltracker.com/story/show/168080405
 
### Description of PR
Added Country & City columns to the Article model.  Also added validation tests.  These back-end columns will be compared against a front-end geotag so that articles based on the user's location will appear on the main page.

### Screenshots
N/A

### What did we learn?
This is a small chore, no new material here.

### CODE REVIEW

File: spec/requests/api/v0/uni_can_pay_for_subscription_spec.rb [hunk 1]
```diff
@@ -8,6 +8,7 @@
   let(:university) { create(:user, role: :university) }
   let(:credentials) { university.create_new_auth_token }
   let(:headers) { { HTTP_ACCEPT: 'application/json' }.merge!(credentials) }
+
```
Comment (julieahlquist) [inline spec/requests/api/v0/uni_can_pay_for_subscription_spec.rb]: Empty row

**Review by julieahlquist — APPROVED** [review]

File: spec/requests/api/v0/uni_can_pay_for_subscription_spec.rb (hunk 1 above)
Comment (sethnejame) [reply spec/requests/api/v0/uni_can_pay_for_subscription_spec.rb]: Thanks!  I just talked to Dash and I think we may have to scrap this PR. . .Thomas had a discussion with her this weekend and said we were approaching this the wrong way.  To be continued. . .I'll fix this meanwhile.

Comment (tochman) [reply spec/requests/api/v0/uni_can_pay_for_subscription_spec.rb]: Well, the big question is why you add this column? How will that fit into the bigger picture?

Comment (sethnejame) [reply spec/requests/api/v0/uni_can_pay_for_subscription_spec.rb]: @tochman I'm trying to get the team on board with this now (we need to fix our user story and vote) but basically we'll use something similar to your demo to get coords to pop up on the main page. . .use OpenCage to turn those coords into City/Country . . .then sort the articles on the main page under something like "Local Research" where the front end coords will be compared against the country/city column on the back end and be displayed.

**Comment by tochman** [comment]
The very least I would do here is to sen that information off to the client (serialize it).  Of course, adding a spec for it is also appropriate.

**Pull request closed without merging**